Source/Tools/FEXLoader/LinuxSyscalls/GdbServer.h

```diff
@@ -93,6 +93,7 @@ class GdbServer {
     std::array<bool, FEXCore::SignalDelegator::MAX_SIGNALS + 1> PassSignals{};
     uint32_t CurrentDebuggingThread{};
     int ListenSocket{};
+    fextl::string GdbUnixSocket{};
```

Comment:
Subjectively unix sockets seem like a rare thing to be used. I typically think of bsd sockets first when reading the term, which makes it surprising to see it `string`-typed. Calling this variable `GdbUnixSocketPath` would make it a bit less surprising.